RetailCoder.VBE/UnitTesting/IAssert.cs

```diff
@@ -10,40 +10,40 @@ public interface IAssert
         /// </summary>
         /// <param name="condition">Any Boolean value or expression.</param>
         /// <param name="message">An optional message to display if the assertion fails.</param>
-        void IsTrue(bool condition, string message = null);
+        void IsTrue(bool condition, string message = "");
```

Comment:
Pretty sure it's required for using `String.Format()`, which makes the localization strings easier to deal with.